packages/portalnetwork/src/networks/network.ts

```diff
@@ -398,7 +414,7 @@ export abstract class BaseNetwork extends EventEmitter {
     }
   }
 
-  handlePing = async (src: INodeAddress, id: bigint, pingMessage: PingMessage) => {
+  handlePing = async (src: INodeAddress, id: bigint, pingMessage: PingMessage, _version: Version) => {
```

Comment:
Why are we adding this here?  Are you just future proofing since we might need it some day?

Reply:
Yes.  we could remove it if that is confusing

Reply:
It's fine. Just wasn't sure what it was doing there